back/src/common/constants/WASTES.ts

```diff
@@ -5642,5 +5642,8 @@ export function toWasteTree(
 }
 
 export function isDangerous(wasteCode: string): boolean {
+  if (!wasteCode) {
+    return false;
+  }
   return wasteCode.endsWith("*");
```

Comment:
```suggestion
  return wasteCode?.endsWith("*");
```

nitpick : ça ne marcherait pas aussi

Reply:
ça semble marcher aussi mais je préfère renvoyer un booléen comme c'est défini dans le typage de la fonction